src/MetricsMsg.chpl

```diff
@@ -181,6 +186,10 @@ module MetricsMsg {
             return this.measurements.size;
         }
 
+        proc add(metric: string, measurement: real) throws {
```

Comment:
Not sure if I am right or not, but would it be better to name this function `update` or `updateTotal`? I think this could be confused with set in some cases if someone isn't paying close attention.

Reply:
@Ethan-DeBandi99 um, I understand where you're coming from, but no. the add function corresponds to adding a measurement to a metric value as opposed to setting the metric value. To me update a synonym for set

Reply:
however, chapel doc comments would make sense here. I will add 'em

Reply:
Sounds good to me. Once those get added, I will approve.